### DISCUSSION TITLE
Migrating from Zenchants to EnchantmentPlus

### DISCUSSION THREAD

**Discussion by NightScythe1** [Geolykt/EnchantmentsPlus#40] [Q&A]
Hey! I've been using Zenchantments for just over a year, and they've become quite bugged on my 1.16.5 server now. What's the best way to switch from that to this plugin? Would i need to wipe zenchants data & configs and recreate it for this plugin? And will existing zenchants be lost? Thanks in advance!

**Comment by Geolykt** [comment]
I would back up and just swap the jars, of course you'd need to reconfigure what needs to be reconfigured, but I guess that should be easy to do.
You should stick with the lore gatherer though (it's the default, so don't worry about that).

It is important to note that the plugin names are different, so they are not compatible 1:1, otherwise there is no difference that should pose an issue
